Geometry/HGCalCommonData/plugins/dd4hep/HGCalDD4HepHelper.h

```diff
@@ -0,0 +1,11 @@
+#ifndef Geometry_HGCalCommonData_HGCalDD4HepHelper_h
+#define Geometry_HGCalCommonData_HGCalDD4HepHelper_h
+#include "DD4hep/DD4hepUnits.h"
+
+namespace HGCalDD4HepHelper {
+  template <class NumType>
+  inline constexpr NumType convert2mm(NumType length) {
+    return (length / dd4hep::mm);
+  }
```

Comment:
This function is essentially the same as
https://github.com/cms-sw/cmssw/blob/2d67e1eb50c1f48446f1de201bda6eb779c6efc1/Geometry/HcalAlgo/plugins/dd4hep/HcalDD4HepHelper.h#L5-L7
(which, by the way, should be declared `inline`). How about defining the function once instead of duplicating it per subdetector?

Reply:
Best may be to make this in DetectorDescription/DDCMS and use it from there. I shall make first a PR for DetectorDescription/DDCMS and then change boy HCAL and HGCAL